### PR TITLE
Update supervisor-and-application.markdown

### PR DESCRIPTION
**Pull request by chareice**
context is a map,not a function.

### CODE REVIEW

File: getting-started/mix-otp/supervisor-and-application.markdown [hunk 1]
```diff
@@ -70,12 +70,12 @@ We also need to update our tests to give a name when starting the registry. Repl
 
 ```elixir
 setup context do
-  {:ok, registry} = KV.Registry.start_link(context.test)
+  {:ok, registry} = KV.Registry.start_link(context[:test])
```
Comment (josevalim) [inline getting-started/mix-otp/supervisor-and-application.markdown]: `context` is a map and you can access fields of a map using the `map.field` syntax. :)

**Comment by josevalim** [comment]
Thank you but the current code is also correct as is. :)

**Comment by eksperimental** [comment]
Shouldn't this be mentioned in `Kernel.SpecialForms` `./2` documentation?

**Comment by josevalim** [comment]
@eksperimental surely :D

**Comment by chareice** [comment]
@josevalim Thanks，since i'm a beginner in elixir😅

**Pull request closed without merging**